fluentd/configs.d/filter-post-z-retag-two.conf

```diff
@@ -4,12 +4,16 @@
   @label @OUTPUT
 </match>
 
-<match journal.** system.var.log** **_default_** **_openshift_** **_openshift-infra_**>
+# Remove audit logs where type != virt-control
+@include filter-exclude-audit.conf
+
+<match journal.** system.var.log** **_default_** **_openshift_** **_openshift-infra_** audit.log**>
   @type rewrite_tag_filter
   @label @OUTPUT
   rewriterule1 message .+ output_ops_tag
   rewriterule2 MESSAGE .+ output_ops_tag
-  rewriterule3 log .+ output_tag_tag
+  rewriterule3 log .+ output_ops_tag
+  rewriterule4 message !.+ output_ops_tag
```

Comment:
We want to retag all the logs tagged as "journal.** system.var.log** **_default_** **_openshift_** **_openshift-infra_** audit.log**".   Could it be done with one rule instead of 4?

Reply:
It could be done with 1 rule if there was a field that was guaranteed to be present in every matching record.  I don't think we can make that guarantee, so we have to use 2 rules:
```
    rewriterule1 message .+ output_ops_tag
    rewriterule2 message !.+ output_ops_tag
```